dbt_meshify/dbt_projects.py

```diff
@@ -19,17 +20,33 @@ class LocalDbtProject:
     def __init__(self, relative_path_to_project: str) -> None:
         self.relative_path_to_project = relative_path_to_project
         self.manifest = self.get_project_manifest()
+        self.subprojects = []
     
     def path_to_project(self) -> str:
         return os.getcwd() + '/' + self.relative_path_to_project
```

Comment:
Let's consider turning this into a property. Along the way, we should leverage `os.Pathlike` objects instead of strings for better cross-platform support.

```suggestion
    @property
    def path(self) -> os.Pathlike:
        return os.path.join(os.getcwd(), self.relative_path_to_project)
```

Then, in future calls to this property would just be

```python
self.path
```

or 

```python
project.path
```

instead of calling the `path_to_project()` method